### PR TITLE
Logic context tests

### PR DESCRIPTION
**Pull request by fckt**
Adding coverage for context for logic.
A part of https://github.com/nearprotocol/nearcore/issues/1412

### CODE REVIEW

File: runtime/near-vm-logic/tests/test_context.rs [hunk 1]
```diff
@@ -0,0 +1,113 @@
+mod fixtures;
+use near_vm_logic::mocks::mock_external::MockedExternal;
+use near_vm_logic::mocks::mock_memory::MockedMemory;
+use near_vm_logic::{Config, VMContext, VMLogic};
+
+pub fn create_context() -> VMContext {
+    VMContext {
+        current_account_id: "alice".to_owned(),
```
Comment (evgenykuzyakov) [inline runtime/near-vm-logic/tests/test_context.rs]: please switch `to_owned` to `to_string` for strings

File: runtime/near-vm-logic/tests/test_context.rs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+);
+decl_test_bytes!(
+    test_signer_account_pk,
+    signer_account_pk,
+    create_context().signer_account_pk.as_slice()
+);
+
+decl_test_bytes!(test_random_seed, random_seed, create_context().random_seed.as_slice());
+
+decl_test_bytes!(test_input, signer_account_pk, create_context().input.as_slice());
```
Comment (evgenykuzyakov) [inline runtime/near-vm-logic/tests/test_context.rs]: Why is `signer_account_pk` here?

File: runtime/near-vm-logic/tests/test_context.rs (hunk 1 above)
Comment (fckt) [reply runtime/near-vm-logic/tests/test_context.rs]: I don't have any strong opinion on this really, but there is some arguments for using `to_owned`:
https://users.rust-lang.org/t/to-string-vs-to-owned-for-string-literals/1441/6

File: runtime/near-vm-logic/tests/test_context.rs (hunk 2 above)
Comment (fckt) [reply runtime/near-vm-logic/tests/test_context.rs]: thanks!

**Pull request merged**